modules/net-lb-app-ext/self-signed-certs.tf

```diff
@@ -0,0 +1,46 @@
+/**
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+resource "tls_private_key" "private_keys" {
+  for_each  = toset(var.ssl_certificates.self_signed_configs)
+  algorithm = "RSA"
+  rsa_bits  = 4096
```

Comment:
We only support [RSA-2048 or ECDSA P-256](https://cloud.google.com/load-balancing/docs/ssl-certificates/self-managed-certs#create-key-and-cert) encryption algorithms.

Reply:
ah true, let me change that

Reply:
done